packages/commerce-sdk-react/src/test-utils.tsx

```diff
@@ -67,6 +67,51 @@ type NockBackOptions = {
     mode?: nock.BackMode
 }
 
+export const mockAuthCalls = () => {
```

Comment:
Since we moved `mockAuthCalls` here in the global scope, it's it safe to remove these lines to avoid duplicated code:
https://github.com/SalesforceCommerceCloud/pwa-kit/blob/97d2b96ae7e7cf262544bdb964623532710f0eb7/packages/commerce-sdk-react/src/test-utils.tsx#L134-L177

Reply:
Ah, thanks for the catch!